modules/streams/src/main/scala/dev/profunktor/redis4cats/pubsub/PubSub.scala

```diff
@@ -55,17 +55,17 @@ object PubSub {
   def mkPubSubConnection[F[_]: ConcurrentEffect: ContextShift: Log, K, V](
       client: RedisClient,
       codec: RedisCodec[K, V]
-  ): Stream[F, PubSubCommands[Stream[F, *], K, V]] =
-    Stream.resource(mkBlocker[F]).flatMap { blocker =>
+  ): Resource[F, PubSubCommands[Stream[F, *], K, V]] =
+    mkBlocker[F].flatMap { blocker =>
       val (acquire, release) = acquireAndRelease[F, K, V](client, codec, blocker)
       // One exclusive connection for subscriptions and another connection for publishing / stats
       for {
-        state <- Stream.eval(Ref.of(Map.empty[K, Topic[F, Option[V]]]))
-        sConn <- Stream.bracket(acquire)(release)
-        pConn <- Stream.bracket(acquire)(release)
-        subs <- Stream.emit(new LivePubSubCommands[F, K, V](state, sConn, pConn, blocker))
-      } yield subs
-
+        state <- Resource.liftF(Ref.of[F, Map[K, Topic[F, Option[V]]]](Map.empty))
+        sConn <- Resource.make(acquire)(release)
+        pConn <- Resource.make(acquire)(release)
+      } yield {
```

Comment:
nitpick: you can remove the braces after `yield`.

Reply:
done